bugbug/bug_features.py

```diff
@@ -258,6 +259,23 @@ def __call__(self, bug, **kwargs):
         )
 
 
+class has_image_attachment_at_bug_creation(object):
+    def __call__(self, bug, **kwargs):
+        bug_creation_time = dateutil.parser.parse(bug["creation_time"])
+        return any(
+            "image" in attachment["content_type"]
+            and dateutil.parser.parse(attachment["creation_time"]) == bug_creation_time
```

Comment:
I'd say that attachment["creation_time"] == bug_creation_time is enough

Reply:
Done with 2cecd1d.